server/src/test/java/org/elasticsearch/action/bulk/TransportShardBulkActionTests.java

```diff
@@ -30,136 +30,154 @@
-import org.elasticsearch.index.mapper.MapperService;
 import org.elasticsearch.index.mapper.Mapping;
+import org.elasticsearch.index.mapper.MetadataFieldMapper;
+import org.elasticsearch.index.seqno.SequenceNumbers;
 import org.elasticsearch.index.shard.IndexShard;
 import org.elasticsearch.index.shard.IndexShardTestCase;
 import org.elasticsearch.index.shard.ShardId;
 import org.elasticsearch.index.translog.Translog;
 import org.elasticsearch.rest.RestStatus;
+import org.junit.After;
+import org.junit.Before;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.Collections;
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicInteger;
-import java.util.function.LongSupplier;
 
 import static org.elasticsearch.action.bulk.TransportShardBulkAction.replicaItemExecutionMode;
+import static org.elasticsearch.test.ClusterServiceUtils.createClusterService;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.Matchers.arrayWithSize;
 import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.nullValue;
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.anyBoolean;
 import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 public class TransportShardBulkActionTests extends IndexShardTestCase {
 
     private final ShardId shardId = new ShardId("index", "_na_", 0);
     private final Settings idxSettings = Settings.builder()
-            .put("index.number_of_shards", 1)
-            .put("index.number_of_replicas", 0)
-            .put("index.version.created", Version.CURRENT.id)
-            .build();
+        .put("index.number_of_shards", 1)
+        .put("index.number_of_replicas", 0)
+        .put("index.version.created", Version.CURRENT.id)
+        .build();
 
     private IndexMetaData indexMetaData() throws IOException {
         return IndexMetaData.builder("index")
-                .putMapping("_doc",
-                        "{\"properties\":{\"foo\":{\"type\":\"text\",\"fields\":" +
-                                "{\"keyword\":{\"type\":\"keyword\",\"ignore_above\":256}}}}}")
-                .settings(idxSettings)
-                .primaryTerm(0, 1).build();
+            .putMapping("_doc",
+                "{\"properties\":{\"foo\":{\"type\":\"text\",\"fields\":" +
+                    "{\"keyword\":{\"type\":\"keyword\",\"ignore_above\":256}}}}}")
+            .settings(idxSettings)
+            .primaryTerm(0, 1).build();
+    }
+
+    private ClusterService clusterService;
```

Comment:
forgot to remove this in the commit I added. This is not needed anymore by tests

Reply:
removed